.github/workflows/notebook-controller-images-updater.yaml

```diff
@@ -31,23 +51,32 @@ jobs:
       with:
         ref: ${{ env.BRANCH_NAME }}
 
+    - name: Debug Inputs
+      run: |
+        echo "generate-pr: ${{ env.GENERATE_PR }}"
+        echo "branch-name: ${{ env.BRANCH_NAME }}"
+        echo "organization: ${{ env.REPO_OWNER }}"
+        echo "temp-branch-name: ${{ env.TEMP_UPDATER_BRANCH }}"
+
+
     - name: Checkout new branch
       run: |
-       echo ${{ env.TEMP_UPDATER_BRANCH }}
-       git checkout -b ${{ env.TEMP_UPDATER_BRANCH }}
-       git push --set-upstream origin ${{ env.TEMP_UPDATER_BRANCH }}
+        echo ${{ env.TEMP_UPDATER_BRANCH }}
+        git fetch origin
+        git checkout -b ${{ env.TEMP_UPDATER_BRANCH }} origin/${{ env.BRANCH_NAME }}
+        git push --set-upstream origin ${{ env.TEMP_UPDATER_BRANCH }}
 
     - name: Configure Git
```

Comment:
Minor nit - but conceptually I would think we'd want to configure `git` **BEFORE** we start doing any `git` operations.. and we are using `git` to checkout the branch in the prior job...